wv will fail pkgconf test pipeline

If the pkgconf test pipeline is added to wv it will fail in the following way:

```
2024/11/12 23:16:10 INFO running step "test/pkgconf"
2024/11/12 23:16:10 WARN + '[' -d /home/build ] uses=test/pkgconf
2024/11/12 23:16:10 WARN + cd /home/build uses=test/pkgconf
2024/11/12 23:16:10 WARN + exit 0 uses=test/pkgconf
2024/11/12 23:16:10 INFO running step "pkgconf build dependency check" uses=test/pkgconf
2024/11/12 23:16:10 WARN + '[' -d /home/build ] uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:10 WARN + cd /home/build uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:10 WARN + basename /home/build/melange-out/wv-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:10 WARN + dev_pkg=wv-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:10 WARN + cd / uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:10 WARN + apk info -L wv-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:10 WARN + grep '\.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:10 WARN WARNING: opening /home/user/src/wolfi-os/packages: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:10 WARN WARNING: opening from cache https://packages.wolfi.dev/os: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:10 WARN + grep -q ^Name: usr/lib/pkgconfig/wv-1.0.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:10 WARN + basename usr/lib/pkgconfig/wv-1.0.pc .pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:10 WARN + lib_name=wv-1.0 uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:10 WARN + echo usr/lib/pkgconfig/wv-1.0.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:10 WARN + grep -q '^usr/lib/pkgconfig/wv-1.0.pc$\|^usr/share/pkgconfig/wv-1.0.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:10 WARN + pkgconf --exists wv-1.0 uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:10 WARN + pkgconf --print-errors --exists wv-1.0 uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:10 WARN Package libgsf-1 was not found in the pkg-config search path. uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:10 WARN Perhaps you should add the directory containing `libgsf-1.pc' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:10 WARN to the PKG_CONFIG_PATH environment variable uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:10 WARN Package 'libgsf-1', required by 'wv-1.0', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:16:10 INFO pod b80fe293a23ed7df0492a47a8cb95fd9ce285cd47755f90da7bad0419a94a464 terminated
2024/11/12 23:16:10 INFO pod d3ce526b092542ff8e9494ad208655f52c306fcfce64473cdfdc1209850ab041 terminated
2024/11/12 23:16:10 INFO ERROR: failed to test package. the test environment has been preserved:
2024/11/12 23:16:10 INFO   workspace dir: /home/user/tmp/melange-workspace-2923610687
2024/11/12 23:16:10 INFO   guest dir: /home/user/tmp/melange-guest-4034936936
2024/11/12 23:16:10 ERRO failed to test package: unable to run pipeline: unable to run pipeline: unable to run pipeline: task exited with code 1
make: *** [Makefile:147: test/wv] Error 1
```

--- a/wv.yaml
+++ b/wv.yaml
@@ -2,7 +2,7 @@
 package:
   name: wv
   version: 1.2.9
-  epoch: 2
+  epoch: 3
   description: A library that can load and parse Word 2000, 97, 95 and 6 file formats
   copyright:
     - license: GPL-2.0-or-later
@@ -43,8 +43,12 @@ subpackages:
       - uses: split/dev
     dependencies:
       runtime:
+        - libgsf-dev
         - wv
     description: wv dev
+    test:
+      pipeline:
+        - uses: test/pkgconf
 
   - name: wv-doc
     pipeline:

